pkg/kubelet/util/cache/version_cache.go

```diff
@@ -17,14 +17,12 @@ limitations under the License.
 package cache
 
 import (
-	"fmt"
 	"sync"
 	"time"
 
 	"github.com/golang/glog"
 
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
-	"k8s.io/kubernetes/pkg/util/wait"
 )
 
 type VersionCache struct {
```

Comment:
@yujuhong Looks like the `expiration_cache` is a little too powerful (complex). :)